tests/fixtures/MakeReactApp/tests/GeneratedReactTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Tests;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class GeneratedReactTest extends WebTestCase
+{
+    public function testReactApp()
+    {
+        $client = self::createClient();
+        $client->request('GET', '/app-react');
+
+        // dump($client->getResponse()->getStatusCode());
```

Comment:
This test is failing, i don't know why, controller exists with the right route in `tmp/cache`
Also, we need to install nodeJs dynamically with dependencies required for a react app. If you know if a method exists in the test or how to create it in th best way.